core/model/modx/processors/security/group/getnodes.class.php

```diff
@@ -131,7 +131,7 @@ public function prepareGroup(modUserGroup $group) {
         $c->limit(1);
         $count = $this->modx->getCount('modUserGroup', $c);
         return array(
-            'text' => $group->get('name').' ('.$group->get('id').')',
+            'text' => htmlentities($group->get('name'), ENT_QUOTES, 'UTF-8') . ' ('.$group->get('id') . ')',
```

Comment:
I'm not sure if this matters, but should UTF-8 be hard-coded here, or is it preferable to rely on the defined `modx_charset`?